cypress/integration/my_cards_spec.js

```diff
@@ -12,8 +12,9 @@ describe('My Cards', () => {
 
     describe('player clicks on a card', () => {
         it("its highlighted by being moved upwards on the y axis", () => {
-            cy.get('[data-cy="my-cards"] [data-cy="card"]').first().click();
-            cy.get('[data-cy="my-cards"]').children().first().should('have.css', "transform", "matrix(1, 0, 0, 1, 0, -30)");
+            cy.get('[data-cy="my-cards"] [data-cy="card"]').first().click().then(() => {
```

Comment:
You might be able to use async/await but if this approach works, there's nothing wrong with it.

Reply:
I saw it done like this on the cypress website

Reply:
To elaborate, you'd need to make the function `async` and do something like:
```
it("its highlighted by being moved upwards on the y axis", async () => {
    await cy.get('[data-cy="my-cards"] [data-cy="card"]').first().click();
    cy.get('[data-cy="my-cards"]').children().first().should('have.css', "transform", "matrix(1, 0, 0, 1, 0, -30)");
```
I'm not sure if you'd need to use `done` somewhere in here.

Reply:
Oh I see, okay

Reply:
I just realised I hadn't seen your reply when I was writing mine.